crates/ruff_linter/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -687,11 +726,10 @@ fn handle_single_name(checker: &mut Checker, expr: &Expr, value: &Expr) {
         expr.range(),
     );
 
-    let node = value.clone();
-    diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
-        checker.generator().expr(&node),
-        expr.range(),
-    )));
+    diagnostic.set_fix(Fix::safe_edits(
```

Comment:
@MichaReiser 

```suggestion
    diagnostic.set_fix(Fix::unsafe_edits(
```

Should `unsafe_edits` be used in case `argvalues` contains comments?

```python
pytest.mark.parametrize(
    "param",
    [
        (
            # hello
            1,
        ),
        (2,),
    ],
)
```

Reply:
If the comment gets removed, then yes, the fix should be marked as unsafe

Reply:
Got it. I'll make this change.